Source/quests.h

```diff
@@ -58,6 +58,13 @@ struct Quest {
 	uint8_t _qvar2;
 
 	bool IsAvailable();
+
+	void activateMultiStageQuest(uint8_t quest_stage = 1)
+	{
+		_qactive = QUEST_ACTIVE;
+		_qlog = true;
+		_qvar1 = quest_stage;

```

Comment:
The paramter feels a bit odd, since it is only used by two quests. I think it would be clearer to simply update it immediately after calling this.

Reply:
were you thinking we make this function always set qvar1 to 1, then add the assignment to 2 immediately after for the two quests which use it? Or just make this function `activate() { _qactive = ACTIVE; _qlog=true; }` and set qvar1 explicitly for every use?

Reply:
> were you thinking we make this function always set qvar1 to 1, then add the assignment to 2 immediately after for the two quests which use it?

Yeah that was my thinking. But I haven't had time to sit down and read the code so it might be miss guided if this isn't a common pattern

Reply:
I'd actually done it that way originally then decided it looked nicer to have the default argument instead. That said it's an inline function anyway so compilers should be able to get rid of the first assignment to qvar1 pretty easily.